app/src/release/java/io/github/droidkaigi/confsched2018/di/BuildTypeBasedNetworkModule.kt

```diff
@@ -0,0 +1,16 @@
+package io.github.droidkaigi.confsched2018.di
+
+import dagger.Module
+import dagger.Provides
+import dagger.multibindings.IntoSet
+import okhttp3.Interceptor
+import javax.inject.Singleton
+
+@Module
+internal class BuildTypeBasedNetworkModule {
+
+    @NetworkLogger @Singleton @Provides @IntoSet
+    fun provideNetworkLogger(): Interceptor = HttpLoggingInterceptor().apply {
+        level = HttpLoggingInterceptor.Level.NONE
```

Comment:
We selected BASIC for release. What do you think? 🐇 
https://github.com/DroidKaigi/conference-app-2018/pull/50#discussion_r160300524